src/jdk.compiler/share/classes/com/sun/tools/javac/parser/DocCommentParser.java

```diff
@@ -154,32 +163,48 @@ char peekChar() {
     }
 
     protected List<DCTree> blockContent() {
-        return blockContent(Phase.BODY);
+        return content(Phase.BODY);
     }
 
     /**
-     * Read block content, consisting of text, html and inline tags.
-     * Terminated by the end of input, or the beginning of the next block tag:
-     * that is, @ as the first non-whitespace character on a line.
+     * Reads "rich text" content, consisting of text, html and inline tags,
+     * according to the given {@code phase}.
+     *
+     * Inline tags are only recognized in {@code BODY} and {@code INLINE}
+     * phases, and not in {@code PREAMBLE} and {@code POSTAMBLE} phases.
+     *
+     * The end of the content is dependent on the phase:
+     *
+     * <ul>
+     * <li>{@code PREAMBLE}: the appearance of {@code <body>} (or {@code <main>}),
+     *      as determined by {@link #isEndPreamble()}
+     * <li>{@code BODY}: the beginning of a block tag, or when readung from
+     *      an HTML file, the appearance of {@code </main>} (or {@code </body>},
+     *       as determined by {@link #isEndBody()}
```

Comment:
Lost trailing )

Reply:
All look OK to me; if you still think there is an issue, can you be more specific?